contrib/xfce4/template.py

```diff
@@ -0,0 +1,50 @@
+pkgname = "xfce4"
+pkgver = "4.18"
+pkgrel = 0
+build_style = "meta"
+depends = [
+    "adw-gtk3",
+    "adw-xfwm4",
+    "fonts-dejavu",
```

Comment:
same with font

Reply:
from xfce4-settings's xsettings.xml:
```xml
    <property name="FontName" type="string" value="Sans 10"/>
    <property name="MonospaceFontName" type="string" value="Monospace 10"/>
```
given the lack of an explicit font family choice, we could sensibly remove this entirely, and people would either install an appropriate font package manually or if they have `base-desktop` they would get `fonts-dejavu` for free